src/Explorer/Tabs/TriggerTabContent.tsx

```diff
@@ -290,7 +298,6 @@ export class TriggerTabContent extends Component<TriggerTab, ITriggerTabContentS
   };
 
   render(): JSX.Element {
-    useCommandBar.getState().setContextButtons(this.getTabsButtons());
```

Comment:
Setting context buttons during render was the source of this issue. Even when the tab is inactive, the React component is rendered (just with `display: none` in CSS) and this stomps on any context buttons set by the _actually_ active tab.

Reply:
nit: would be great to add such findings to the commit message or as a comment in code making it easier to understand the change later (next time)